operatorcourier/cli.py

```diff
@@ -59,7 +59,7 @@ def verify(self):
         parser.add_argument('source_dir', help='Path of your directory of yaml '
                             'files to bundle. Either set this or '
                             'use the files argument for bundle data.')
-        parser.add_argument('--ui_validate_io',
+        parser.add_argument('--ui-validate-io',
```

Comment:
This should be a separate pr, since we are introducing a breaking change on a released feature.